docs/quick-start.md

```diff
@@ -3,22 +3,373 @@ title: Getting Started
+    name: example-gcp-credentials
+    namespace: crossplane-system
+    key: credentials.json
+```
+
+Save the above `Provider` as `provider.yaml`, save your Google Application
+Credentials as `credentials.json`, then run:
+
+```bash
+kubectl -n crossplane-system create secret example-gcp-credentials --from-file=credentials.json
+kubectl apply -f provider.yaml
+```
+
+Crossplane can now manage your GCP project! Your service account will need the
+CloudSQL Admin role for this guide. Check out GCP's [Getting Started With
+Authentication] guide if you need help creating a service account and
+downloading its `credentials.json` file, and Crossplane's [GCP provider
+documentation] for detailed instructions on setting up your project and service
+account permissions.
+
+## Provision a MySQL Instance
+
+GCP provides MySQL databases using [CloudSQL] instances. Crossplane uses a
+resource and claim pattern to provision and manage cloud resources like CloudSQL
+instances - if you've ever used [persistent volumes in Kubernetes] you've seen
+this pattern before. The simplest way to start using a new MySQL instance on GCP
+is to provision a `CloudsqlInstance`, then claim it via a `MySQLInstance`. We
+call this process _static provisioning_.
+
+
+```yaml
+apiVersion: database.gcp.crossplane.io/v1beta1
+kind: CloudsqlInstance
+metadata:
+  name: example-cloudsql-instance
+spec:
+  providerRef:
+    name: example-provider
+  writeConnectionSecretToRef:
+    name: example-cloudsql-connection-details
+    namespace: crossplane-system
+  forProvider:
+    databaseVersion: MYSQL_5_6
+    region: us-west2
+    settings:
+      tier: db-n1-standard-1
+      dataDiskType: PD_SSD
+      dataDiskSizeGb: 10
+      ipConfiguration:
+        ipv4Enabled: true
+```
+
+First we create a CloudSQL instance. Save the above as `cloudsql.yaml`, then
+apply it using `kubectl`:
+
+```bash
+kubectl apply -f cloudsql.yaml
+```
+
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: example-mysql-claim
+spec:
+  resourceRef:
+    apiVersion: database.gcp.crossplane.io/v1beta1
+    kind: CloudsqlInstance
+    name: example-cloudsql-instance
+  writeConnectionSecretToRef:
+    name: example-mysql-connection-details
+```
+
+Crossplane is now creating the `CloudsqlInstance`! Before we can use it, we need
+to claim it. Save the above as `mysql.yaml`, and once again apply it via
+`kubectl`:
+
+```bash
+kubectl --namespace default apply -f mysql.yaml
+```
+
+In Crossplane cloud provider specific resources like the `CloudsqlInstance` we
+created above are called _managed resources_. They're considered infrastructure,
+like a Kubernetes `Node` or `PersistentVolume`. Managed resources exist at the
+cluster scope (they're not namespaced) and let you specify nitty-gritty provider
+specific configuration details. Managed resources that have reached `v1beta1`
+support at least 90% of their underlying cloud provider resource's configuration
```

Comment:
is providing exact percentage necessary?

Reply:
maybe "a large portion" or "majority" instead?

Reply:
We don't actually have an exact percentage requirement, though we may as part of https://github.com/crossplaneio/crossplane/issues/933.

What I want to say here is that v1beta1 resources support _all_ of their underlying cloud provider resource's configuration. The catch is that there are a few edge cases such that we support 98/100 configuration options, so I can't truthfully say that. A "large portion" or "majority" feels too vague to me - I could read that as supporting 2/3rds of what CloudSQL could do when in reality we support almost all of what CloudSQL can do.

Reply:
I ended up just using the terminology we use - "high fidelity". I feel like this is a common enough term that it will communicate what I intended.